spec.html

```diff
@@ -1110,30 +1110,898 @@ <h1>Well-Known Symbols</h1>
+          </emu-alg>
+        </emu-clause>
+
+        <emu-clause id="sec-numeric-types-bigint-bitwiseNOT">
+          <h1>BigInt::bitwiseNOT ( _x_ )</h1>
+          <p>The abstract operation BigInt::bitwiseNOT with an argument _x_ of type BigInt returns the one's complement of _x_; that is, -_x_ - 1.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-numeric-types-bigint-exponentiate">
+          <h1>BigInt::exponentiate ( _base_, _exponent_ )</h1>
+          <emu-alg>
+            1. If _exponent_ &lt; *0n*, throw a *RangeError* exception.
+            1. If _base_ is *0n* and _exponent_ is *0n*, return *1n*.
+            1. Return the BigInt value that represents the mathematical value of _base_ raised to the power _exponent_.
+          </emu-alg>
+        </emu-clause>
+
+        <emu-clause id="sec-numeric-types-bigint-multiply">
+          <h1>BigInt::multiply ( _x_, _y_ )</h1>
+          <p>The abstract operation BigInt::multiply with two arguments _x_ and _y_ of type BigInt returns the BigInt value that represents the result of multiplying _x_ and _y_.</p>
+          <emu-note>Even if the result has a much larger bit width than the input, the exact mathematical answer is given.</emu-note>
+        </emu-clause>
+
+        <emu-clause id="sec-numeric-types-bigint-divide">
+          <h1>BigInt::divide ( _x_, _y_ )</h1>
+          <emu-alg>
+            1. If _y_ is *0n*, throw a *RangeError* exception.
+            1. Let _quotient_ be the mathematical value of _x_ divided by _y_.
+            1. Return the BigInt value that represents _quotient_ rounded towards 0 to the next integral value.
+          </emu-alg>
+        </emu-clause>
+
+        <emu-clause id="sec-numeric-types-bigint-remainder">
+          <h1>BigInt::remainder ( _n_, _d_ )</h1>
+          <emu-alg>
+            1. If _d_ is *0n*, throw a *RangeError* exception.
+            1. If _n_ is *0n*, return *0n*.
+            1. Let _r_ be the BigInt defined by the mathematical relation _r_ = _n_ - (_d_ &times; _q_) where _q_ is a BigInt that is negative only if _n_/_d_ is negative and positive only if _n_/_d_ is positive, and whose magnitude is as large as possible without exceeding the magnitude of the true mathematical quotient of _n_ and _d_.
+            1. Return _r_.
+          </emu-alg>
+          <emu-note>The sign of the result equals the sign of the dividend.</emu-note>
+        </emu-clause>
+
+        <emu-clause id="sec-numeric-types-bigint-add">
+          <h1>BigInt::add ( _x_, _y_ )</h1>
+          <p>The abstract operation BigInt::add with two arguments _x_ and _y_ of type BigInt returns the BigInt value that represents the sum of _x_ and _y_.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-numeric-types-bigint-subtract">
+          <h1>BigInt::subtract ( _x_, _y_ )</h1>
+          <p>The abstract operation BigInt::subtract with two arguments _x_ and _y_ of type BigInt returns the BigInt representing the difference _x_ minus _y_.</p>
```

Comment:
```suggestion
          <p>The abstract operation BigInt::subtract with two arguments _x_ and _y_ of type BigInt returns the BigInt value that represents the difference _x_ minus _y_.</p>
```
... for consistency with my last commit. (I missed it because I was only looking for "*a* BigInt representing".)